src/Convert/ConvertInterface.php

```diff
@@ -143,10 +143,11 @@ public function toDegreesMinutesSeconds($format = ConvertInterface::DEFAULT_DMS_
      * Convert and format a decimal degree coordinate to decimal minutes coordinate.
      *
      * @param string $format The way to format the DMS coordinate.
+     * @param bool|int $precision Force the precision for the decimal part
      *
      * @return string Converted and formatted string.
      */
-    public function toDecimalMinutes($format = ConvertInterface::DEFAULT_DM_FORMAT);
+    public function toDecimalMinutes($format = ConvertInterface::DEFAULT_DM_FORMAT, $precision = false);
```

Comment:
This is a BC break, isnt it?

Reply:
nope, the default value doesn't change the result. I will change it to null

Reply:
That is true for classes. But is it really true for interfaces?

I'll check when I'm at a computer.

Reply:
Hello @Nyholm,
I rebase this PR, could you check it again please?